### DISCUSSION TITLE
OIDC Support

### DISCUSSION THREAD

**Discussion by daedric7** [blinkospace/blinko#125] [Ideas]
Would it be hard to implement OIDC support ? I'm quite fond of a central user DB where possible :)

**Comment by blinko-space** [comment]
now support in [v0.32.0](https://github.com/blinko-space/blinko/releases/tag/v0.32.0)

**Comment by daedric7** [comment]
I tried it! :)

One flaw i bumped into, is the username.

Instead of something from the upstream oidc provider, it gets to be a random string:

![image](https://github.com/user-attachments/assets/933405c4-cea0-4c8f-a3a7-5783823349d0)

Could this be perhaps some info from oidc ?

Also, Blink allows us to change this informatio:

![image](https://github.com/user-attachments/assets/c0c800ec-1d39-4073-99e8-95cffe9603b8)

Username is not supposed to be changeable by the user when using OIDC. This novertheless goes forward, though  the "new" user is never used:

![image](https://github.com/user-attachments/assets/6d45bd9b-30dd-44fe-8e13-1659b31697f6)
